addon/helpers/d3-array.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import array from 'd3-array';
+
+let allowedMethods = ['min', 'max', 'extent', 'sum', 'mean', 'median', 'variance', 'deviation', 'scan', 'merge', 'pairs', 'permute'];
+export function d3Array([ method, ...args ]) {
+  Ember.assert('this method is not yet supported', Ember.A(allowedMethods).contains(method));
```

Comment:
Not really sure how much this matters since the array is so small, but I wonder if creating a new array instance and doing a `contains` might be a lot of overhead just to do an assert. 

At the very least, it might be worth making sure only one Ember array is ever created. i.e.: 

```js
let allowedMethods = Ember.A(['min', 'max', 'extent', 'sum', 'mean', 'median', 'variance', 'deviation', 'scan', 'merge', 'pairs', 'permute']);
 export function d3Array([ method, ...args ]) {
   Ember.assert('this method is not yet supported', allowedMethods.contains(method));
   ...etc
}
```

Other than this catching my eye, this stuff looks good!